### ISSUE TITLE
Back button is practically invisible when the Event Listing Image's upper-left corner is very dark

### ISSUE THREAD

**Issue by declanslevin** [redbadger/pride-london-app#94]
It might be worth either replacing the "Back" text with an icon, or give it a shadow or backdrop of some kind so that it is always visible regardless of the background image. Currently it blends into the image if the upper-left corner is very dark:
![20180404_174944](https://user-images.githubusercontent.com/33623842/38322146-801b5fc0-3831-11e8-9a53-3ef85b045751.jpg)

**Comment by stevecottle** [comment]
Current designs have a purple header so think this is covered in upcoming stories. @JunBadger could you confirm?

**Comment by JunBadger** [comment]
Yup - that's right - will be the sticky header @stevecottle @declanslevin

**Comment by declanslevin** [comment]
Ok, I'm guessing that should be covered by https://trello.com/c/iW3T4PWb/128-ui-events-listing-view-individual-event-details - I added a link to this issue as a reminder to check it is implemented as part of that work. 

Removing the high priority tag for now
